flycheck.el

```diff
@@ -3893,8 +3895,8 @@ The following PROPERTIES constitute an error level:
         (plist-get properties :compilation-level))
   (setf (get level 'flycheck-overlay-category)
         (plist-get properties :overlay-category))
-  (setf (get level 'flycheck-fringe-bitmap-double-arrow)
-        (plist-get properties :fringe-bitmap))
+  (setf (get level 'flycheck-fringe-bitmaps-double-arrow)
+        (plist-get properties :fringe-bitmaps))
```

Comment:
Here we should check whether the value returned by `plist-get` is a cons, and if not create a cons out of it.